src/ray/util/logging.h

```diff
@@ -126,6 +126,18 @@ enum class RayLogLevel {
 
 #endif  // NDEBUG
 
+#define RAY_CHECK_OP(left, op, right)        \
+  if (const auto &_left_ = (left); true)     \
+    if (const auto &_right_ = (right); true) \
+  RAY_CHECK((_left_ op _right_)) << " " << _left_ << " vs " << _right_
```

Comment:
Indent this line?

Reply:
Not sure why but our format script doesn't want to indent it :(